articles/connections/enterprise/oidc.md

```diff
@@ -51,14 +53,120 @@ If you edit the OIDC Connect Connection, you will see additional configuration o
 
 * **Icon URL**: The icon that will be used in the Login screen for the New Universal Login Experience to identify the connection. In the Classic Universal Login Experience, a default icon will be used.
 
-* **Domain**: The list of email domains that will be routed to this Identity Provider. This is only applicable when using the Classic Universal Login Experience.
+* **Type**: Determines what ODIC options will be used when authenticating with the OIDC provider. You could need to adjust it if the ODIC metadata claims to support both but a specific client only supports Back Channel.
+
+    * Front Channel: Auth0 will use `response_mode=form_post` and `response_type=id_token`. It's the preferred one as it does not require a client secret. 
+    * Back Channel: Auth0 will use the authorization code flow with `response_type=code`.
+
+* **IdP Domains**: The list of email domains that can be authenticated in the Identity Provider. This is only applicable when using [Identifier First](/universal-login/identifier-first) authentication in the Classic Universal Login Experience.
 
-* **Scopes**: The list of OAuth scopes that will be requested when connecting to the identity provider. This will affect the data stored in the user profile. You are required to include at least the 'openid' scope.
+* **Scopes**: The list of OAuth scopes that will be requested when connecting to the identity provider. This will affect the data stored in the user profile. You are required to include at least the 'openid' scope. Note that the connection does not call `/userinfo` endpoint and expects the user claims to be present in the `id_token`.
+
+## Manually configuring Issuer metadata 
+
+If you click `Show Issuer Details` you can see the data returned by the OIDC metadata endpoint and adjust it in case you need to.
```

Comment:
> ...you can see the data returned by the OIDC metadata endpoint...

Perhaps mentioning the Issuer URL (ex: `you can see the data returned from the Issuer URL endpoint`)  here will help the customer mentally connect the concepts?